platform/felib/ui.apps/src/main/content/jcr_root/apps/felib/components/lib/css.html

```diff
@@ -1,3 +1,5 @@
-<sly data-sly-use.model="com.peregrine.felib.models.CSSModel"/><sly data-sly-list="${model.files}">
-/* ${item.path} */
-${item.content @ context='unsafe'}</sly>
\ No newline at end of file
+<sly data-sly-use.model="com.peregrine.felib.models.CSSModel"
+     data-sly-list="${model.files}">
+    /* ${item.path} */
+    ${item.content @ context='unsafe'}
+</sly>
```

Comment:
the choice of formatting was intentional here to make the rendering of the output look better when looking at the generated file in the browser

Reply:
reverted